src/pipeline_eval.py

```diff
@@ -1,4 +1,5 @@
 """Fondant pipeline to evaluate a RAG pipeline."""
+import pyarrow as pa
 from fondant.pipeline import Pipeline
 
 
```

Comment:
Should add a `api_keys: dict = {}` argument in the create_pipeline function. Same in the pipeline_index